packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -898,6 +914,15 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
     UserScrollNotification(metrics: copyWith(), context: context.notificationContext!, direction: direction).dispatch(context.notificationContext);
   }
 
+  /// Dispatches a notification that the contents metrics has changed.
+  ///
+  /// This notification is dispatched when the dimensions of the contents have
+  /// changed. For example, the size of the child changed or new children were
+  /// load when scrolling a long list.
```

Comment:
```suggestion
  /// For example, when the content of a scrollable is altered, making it larger or smaller,
  /// this notification will be dispatched. Similarly, if the size of the window or parent
  /// changes, the scrollable can notify of these changes in dimensions.
```

How does that sound? We should probably also document how this is different from ScrollNotification.